services/user/Branding/ui/src/App.tsx

```diff
@@ -20,7 +20,7 @@ export const App = () => {
     const init = async () => {
         await supervisor.onLoaded();
         supervisor.preLoadPlugins([{ service: "branding" }]);
-        setTimeout(getNetworkName, 1500);
+        await getNetworkName();
```

Comment:
If I understood your other comment correctly, `await` here was not working at some point. 
But I assume from this change that it is now working, yeah?

Reply:
yes